R/coercion.R

```diff
@@ -7,8 +7,16 @@
 #' default methods call the base versions.
 #'
 #' @section Methods:
+#'
+#' **`as.factor()`**
```

Comment:
Would it be better to use `\subsection{}{}` here?

Reply:
Yea, thanks. And maybe that distinguishes things enough that we don't need extra space between generics now. It looks okay to me

![screen shot 2018-10-09 at 10 48 18 am](https://user-images.githubusercontent.com/19150088/46677594-f337c080-cbb0-11e8-9ae1-3d48dbf169ef.png)